source/CardGames.py

```diff
@@ -156,3 +156,27 @@ def dealCards(self, numCards: int, players: "list[Player]"):
   def resetDeck(self):
     self.deck.reset()
     self.deck.shuffle()
+
+def printHands(players: dict):
+    # Show each player's hand
+    for player in players.values():
+        print(f'{player.name}:')
+        player.showHand(True)
+        print()
+
+def player_ages(players):
+    playerAge = []
+    for player in players.values():
+        while True:
```

Comment:
Do you need the while loop?

Reply:
Good point. I will try experimenting without it and see If the code still runs. Thanks for the feedback!